apps/ui/src/components/ui/nav-link.tsx

```diff
@@ -34,13 +34,10 @@ export const NavLink = ({
 }: NavLinkProps) => {
   const pathname = usePathname();
   const isActive = exact ? pathname === href : pathname.startsWith(href);
-
-  variant = "ghost";
```

Comment:
I removed this variant declaration because it was causing storybook to render all the storieswith the `ghost` variant instead of the variant name I was trying to pass to args. I don't think this will cause issues in the next app but wanted to note.